dspace/config/registries/datacite-types.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE dspace-dc-types SYSTEM "dspace-dc-types.dtd">
+
+<!--
+  This document was based on the OpenAIRE4 Guidelines for Literature Repositories
+  https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/index.html
+  -->
+  
+<dspace-dc-types>
+    <dspace-header>
+        <title>Datacite fields definition</title>
+    </dspace-header>
+
+    <dc-schema>
+        <name>datacite</name>
+        <namespace>http://datacite.org/schema/kernel-4</namespace>
+    </dc-schema>
+	
+	
+    <!--
+      OpenAIRE4 Guidelines
+      21 - Geolocation -->	
+    <dc-type>
+        <schema>datacite</schema>
+        <element>geoLocation</element>
+        <scope_note>Spatial region or named place where the data was gathered or about which the data is focused.</scope_note>
```

Comment:
As this seems to be a schema with a single field, I wonder if we should just move this to the `oaire` schema?  Even though I understand OpenAIRE requires the `datacite:geoLocation` field, it is really odd to me to create a schema for one field.  We can always "map" this field to the proper location that OpenAIRE expects in OAI-PMH config.

Reply:
We kept the same schema but move it into oaire file.